examples/knative/powershell/kn-powershell-echo/function.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-ps-echo
+  namespace: vmware-functions
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      containers:
+        - image: projects.registry.vmware.com/veba/kn-ps-echo:latest
```

Comment:
shouldn't this be the one the user creates? at least inconsistent from the lamw image earlier